src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -11436,6 +11436,30 @@ .maxstack  2
 }");
         }
 
+        [Fact]
+        public void Bug2272()
```

Comment:
Consider:  use descriptive name for name of method (even if it's long) and use [WorkItem] attribute to denote the bug id.